client/app/components/sections/Topbar/Topbar.js

```diff
@@ -1,13 +1,54 @@
 import { Component, PropTypes } from 'react';
 import r, { div } from 'r-dom';
 
+import { t } from '../../../utils/i18n';
+import { railsContext } from '../../../utils/PropTypes';
 import css from './Topbar.css';
 
 import Logo from '../../elements/Logo/Logo';
 import SearchBar from '../../composites/SearchBar/SearchBar';
+import Menu from '../../composites/Menu/Menu';
+
+const LABEL_TYPE_MENU = 'menu';
+const LABEL_TYPE_DROPDOWN = 'dropdown';
 
 class Topbar extends Component {
   render() {
+    const menuProps = Object.assign({}, this.props.menu, {
+      key: 'menu',
+      name: t('web.topbar.menu'),
+      identifier: 'Menu',
+      menuLabelType: LABEL_TYPE_MENU,
+      content: this.props.menu.links.map((l) => (
```

Comment:
The `menu` prop isn't required (and should't be), but this line expects it. The menu is only rendered when the prop exists, so the prop check should probably happen when constructing the menu props as well.